app/views/search/_form.html.erb

```diff
@@ -100,37 +112,42 @@ end
         </summary>
         <input id="geodistance-search-field" type="hidden" name="geodistance" value="<%= params[:geodistance] %>">
         <fieldset>
-          <legend>Search within a distance of a geographical point</legend>
+          <legend>Search within a distance of a geographic point</legend>
           <div class="gridband layout-2c">
             <div class="field-wrap grid-item">
-              <label for="geodistance-latitude">
-                Latitude: a decimal between -90.0 and 90.0 (Southern hemisphere is negative)
-              </label>
-              <input type="number" step="0.1" min="-90.0" max="90.0"
+              <label class="geo-label" for="geodistance-latitude">Latitude</label>
+              <input type="number" step="0.000001" min="-90.0" max="90.0"
                      class="field field-text <%= "required" if geodistance_required %>" 
-                     id="geodistance-latitude" name="geodistanceLatitude" placeholder="Latitude"
-                     value="<%= params[:geodistanceLatitude] %>"
-                     <%= 'required="required" aria-required="true"' if geodistance_required %>>
+                     id="geodistance-latitude" name="geodistanceLatitude"
+                     value="<%= params[:geodistanceLatitude] %>" aria-describedby="lat-desc"
+                     <%= 'required="required" aria-required="true"' if geodistance_required %>
+                     aria-describedby="lat-desc" placeholder="E.g., 42.279594">
+              <span class="geo-desc" id="lat-desc">
+                A decimal between -90.0 and 90.0 (Southern hemisphere is negative)
+              </span>
             </div>
             <div class="field-wrap grid-item">
-              <label for="geodistance-longitude">
-                Longitude: a decimal between -90.0 and 90.0 (Western hemisphere is negative)
-              </label>
-              <input type="number" step="0.1" min="-90.0" max="90.0"
-                     class="field field-text <%= "required" if geodistance_required %>" 
-                     id="geodistance-longitude" name="geodistanceLongitude" placeholder="Longitude"
-                     value="<%= params[:geodistanceLongitude] %>"
-                     <%= 'required="required" aria-required="true"' if geodistance_required %>>
+              <label class="geo-label" for="geodistance-longitude">Longitude</label>
+                <input type="number" step="0.000001" min="-180.0" max="180.0"
+                       class="field field-text <%= "required" if geodistance_required %>" 
+                       id="geodistance-longitude" name="geodistanceLongitude"
+                       value="<%= params[:geodistanceLongitude] %>" aria-describedby="long-desc"
+                       <%= 'required="required" aria-required="true"' if geodistance_required %>
+                       aria-describedby="long-desc" placeholder="E.g., -83.732124">
```

Comment:
I see what you did with those lat and long values. Well played :-)

Reply:
😈